packages/os/network-config.service

```
@@ -0,0 +1,20 @@
+[Unit]
+Description=Bottlerocket network configuration system
+# The service attempts to read from this mount
+RequiresMountsFor=/var/lib/bottlerocket
+After=local-fs.target
+Before=wickedd.service network-pre.target
+Wants=network-pre.target
```

Since `wicked` comes after `network-pre.target`, we can just use the target as the synchronization point:
```suggestion
After=local-fs.target
Before=network-pre.target
```

---

That makes sense - I think we still need the `Wants=network-pre.target` though, based what I [read here](https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/).

> Services that want to be run before the network is configured should place Before=network-pre.target and also set Wants=network-pre.target to pull it in. This way, unless there's actually a service that needs to be ordered before the network is up the target is not pulled in, hence avoiding any unnecessary synchronization point.